### PR TITLE
DESCW-828 print mode additions

### PR DESCRIPTION
**Pull request by codewisenate**
- additional core block styling
- print specific styling (eg: tables)
- new classes added to suppress printing (details in readme)

### CODE REVIEW

File: src/styles/public/print.scss [hunk 1]
```diff
@@ -18,9 +24,9 @@
 		* {
 			all: revert;
 			aspect-ratio: unset !important;
-			color: #333 !important;
 			background: none !important;
 			border: none !important;
+			color: #333 !important;
```
Comment (ShawnTurple) [inline src/styles/public/print.scss]: just curious why this color was used #333, design system is #313132 https://developer.gov.bc.ca/Design-System/Colour-Palette

File: src/styles/public/print.scss [hunk 2]
```diff
@@ -223,13 +233,58 @@
...
-					display: block !important;
+				&.wp-block-media-text {
+
+					&__media {
+						display: block !important;
+					}
+				}
+
+				thead {
+					border-bottom: 2px solid #333 !important;
```
Comment (ShawnTurple) [inline src/styles/public/print.scss]: probably should be using a sass variable for colors

**Review by ShawnTurple — APPROVED** [review]

File: src/styles/public/print.scss (hunk 1 above)
Comment (codewisenate) [reply src/styles/public/print.scss]: For print I prefer using a soft black that remains fully neutral.

File: src/styles/public/print.scss (hunk 2 above)
Comment (codewisenate) [reply src/styles/public/print.scss]: I've switched it to the sass $black variable which is not as soft as the previous setting but is also not fully black.

**Review by ShawnTurple — APPROVED** [review]

**Pull request merged**